### PR TITLE
Add 2023 commissioning logbook entries

### PR DESCRIPTION
**Pull request by awegsche**
- some minor corrections to existing entries
- added days 04-06 (validation of new 120cm corrs), 04-10, 04-16 (both Ions)

### CODE REVIEW

File: docs/resources/logbook/2023_lhc.md [hunk 1]
```diff
@@ -10,28 +10,41 @@ The tables below can be sorted by clicking next to the column headers.
...
 [inj_linear_optics_sum]:                    https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-03-29T00%3A00%3A00&dateTo=2023-03-29T23%3A59%3A59&eventToHighlight=3739271 
 [squeezed_linear_optics]:                   https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-01T00%3A00%3A00&dateTo=2023-04-01T23%3A59%3A59&eventToHighlight=3741499 
 [squeezed_linear_optics_sum]:               https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-01T00%3A00%3A00&dateTo=2023-04-01T23%3A59%3A59&eventToHighlight=3741722
+[meas_in_ramp]:                             https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-06T00%3A00%3A00&dateTo=2023-04-06T23%3A59%3A59&eventToHighlight=3744741 
+[meas_in_ramp_sum]:                         https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-06T00%3A00%3A00&dateTo=2023-04-06T23%3A59%3A59&eventToHighlight=3744954
```
Comment (JoschD) [inline docs/resources/logbook/2023_lhc.md:32]: These link to the OP logbook. I am getting "could not retrieve selected events"

File: docs/resources/logbook/2023_lhc.md [hunk 2]
```diff
@@ -10,28 +10,41 @@ The tables below can be sorted by clicking next to the column headers.
...
-*[Ramp and Squeeze Linear Optics]: Measurements and validation of optics from 200cm down to 30cm, including K-Mod. New corrections at 120cm. All others agree well with corrections from the 2023 MD last year. 
+*[Ramp and Squeeze Linear Optics]: Measurements and validation of optics from 200cm down to 30cm, including K-Mod. First new corrections at 120cm, applied from 200cm to 120cm. Bad IP8. All others agree well with corrections from the 2023 MD last year. 
+*[Measurements in the Ramp + Validation]: Measurements in the Ramp, Validation of 120cm optics with corrections done at 04-05 to lower IP8 beating. 
 *[a4 Corrections at 30cm]: Measurement of the f1012 and f1210 RDTs and corrections tests at 30cm. Some K-Mod measurement at 87cm.
 *[NL Studies at Injection]: Measurement of several RDTs with and without landau octupoles with both 2022 and 2023 optics.
+*[Ions Virgin + Local Coupling]: Get beams to end of ramp. Managed only after 2nd try (dump due to kick close to tune). Minor coupling correction.
```
Comment (fsoubelet) [inline docs/resources/logbook/2023_lhc.md:49]: Blunders and stuggles to get to the end of the ramp are irrelevant. What matters for the tooltip is only what was observed and achieved in the shift.

File: docs/resources/logbook/2023_lhc.md [hunk 3]
```diff
@@ -10,28 +10,41 @@ The tables below can be sorted by clicking next to the column headers.
...
 [a4_corrections_30cm]:                      https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-08T00%3A00%3A00&dateTo=2023-04-08T23%3A59%3A59&eventToHighlight=3745378
 [a4_corrections_30cm_sum]:                  https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-08T00%3A00%3A00&dateTo=2023-04-08T23%3A59%3A59&eventToHighlight=3745433
 [nl_studies_injection]:                     https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-08T00%3A00%3A00&dateTo=2023-04-08T23%3A59%3A59&eventToHighlight=3745829
 [nl_studies_injection_sum]:                 https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-08T23%3A00%3A00&dateTo=2023-04-09T07%3A00%3A00&eventToHighlight=3746004
+[ions_virgin_coupl]:                        https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-10T00%3A00%3A00&dateTo=2023-04-10T23%3A59%3A59&eventToHighlight=3746402
```
Comment (fsoubelet) [inline docs/resources/logbook/2023_lhc.md:37]: This link directs to a logbook entry with measurements of the kickgroup. Is there not a better one?

**Review by fsoubelet — CHANGES_REQUESTED** [review]
Needs links fixes and some clarifications. Table formatting is messed up but that's an easy fix for later.

File: docs/resources/logbook/2023_lhc.md (hunk 3 above)
Comment (awegsche) [reply docs/resources/logbook/2023_lhc.md:37]: No, that shift was not well documented

File: docs/resources/logbook/2023_lhc.md (hunk 2 above)
Comment (awegsche) [reply docs/resources/logbook/2023_lhc.md:49]: Ok

File: docs/resources/logbook/2023_lhc.md (hunk 1 above)
Comment (JoschD) [reply docs/resources/logbook/2023_lhc.md:32]: the id is 1081 (OMC logbook) not 322 (OP logbook)

**Comment by awegsche** [comment]
So, problems fixed.

we could optionally change the logbook entry for the first ion shift and it should be good to go

**Comment by awegsche** [comment]
> 

I don't know what you mean by "table formatting messed up" for me it displays perfectly

**Review by fsoubelet — APPROVED** [review]
I meant the formatting of the table code.
I fixed a link that pointed to the wrong day (ramp meas summary) and tweaked the tooltips for ion optics entries.

All links work fine. I'm approving but I'd like to get approval from @JoschD too before we merge.

**Review by JoschD — APPROVED** [review]

**Pull request merged**